scrapy/utils/url.py

```diff
@@ -103,3 +103,16 @@ def guess_scheme(url):
         return any_to_uri(url)
     else:
         return add_http_if_no_scheme(url)
+
+
+def trim_href_attribute(href):
```

Comment:
Though code is easier to read with `trim_html_attribute` name - I think it makes assumptions made by link extractors more clear.